examples/ui-search/src/App.tsx

```diff
@@ -0,0 +1,70 @@
+import { Configuration, EndaomentSdkApi, OrgDto } from '@endaoment/sdk';
+
+import { useState } from 'react';
+import {
+  Container,
+  Input,
+  Button,
+  Stat,
+  StatLabel,
+  StatHelpText,
+  Flex,
+  Box,
+  Heading,
+  Spacer,
+  ButtonGroup,
+  Divider,
+} from '@chakra-ui/react';
+
+const config = new Configuration({ network: 'local' });
+const sdk = new EndaomentSdkApi(config);
+
+function App() {
+  const [search, setSearch] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [searchResultOrgs, setSearchResultOrgs] = useState<OrgDto[]>();
+
+  const handleSearch = async () => {
+    setLoading(true);
+    const orgs = await sdk.searchDeployedOrgs({ name: search });
+    setSearchResultOrgs(orgs);
+    setLoading(false);
+    setSearch('');
+  };
+
+  return (
+    <Container padding={10}>
+      <Flex alignItems="center" gap="2">
+        <Box p="2">
+          <Heading size="md">Art Blocks</Heading>
```

Comment:
Why does this say Art Blocks?

Reply:
Both good points - prepping this up for Friday's demo, which is for Art Blocks. That being said, we should not merge this as is - I'll change this to be the "real" example values